x/tx/textual/internal/textualpb/textual.proto

```diff
@@ -67,23 +67,22 @@ message SignerData {
 // If any of the three structs above is modified, then this Envelope message
 // also needs to be updated.
 message Envelope {
```

Comment:
Is it the doing of `make proto-format`? If not, could you revert?

Reply:
> Is it the doing of `make proto-format`? If not, could you revert?

Yeah, it came from `make proto-all` I didn't see a way to do a single file easily